### PR TITLE
Install menu component 

### PR DESCRIPTION
**Pull request by EmmanuelDrouin**
- Install menu component from the MESG Component repository

### CODE REVIEW

**Comment by EmmanuelDrouin** [comment]
Todo: 
- [ ] Install version 1.0.4 of the Menu component to fix design issues with nuxt-link

**Comment by EmmanuelDrouin** [comment]
@antho1404 ready for review

File: components/Menu.vue [hunk 1]
```diff
@@ -1,313 +1,21 @@
...
-            <nuxt-link :to="links.token" class="top-menu">Token</nuxt-link>
-          </li>
-          <li flex align-center>
-            <Button primary small :to="links.getstarted">Get started</Button>
-          </li>
-        </ul>
-      </div>
-    </nav>
-  </div>
+  <Menu :logo="require('~/assets/MESG-logo-horizontal-purple.svg')" :items="menu" is-nuxt />
```
Comment (antho1404) [inline components/Menu.vue]: Can you just replace the menu where it's used (the layout) like that you can delete this component? It's weird to have a component menu that includes a menu.

**Comment by antho1404** [comment]
The `getStarted` button triggers a full refresh of the page and shouldn't, probably not using a nuxt link.

**Comment by EmmanuelDrouin** [comment]
> The `getStarted` button triggers a full refresh of the page and shouldn't, probably not using a nuxt link.

@antho1404  I found the issue, it come from the component. 
The button is set to an href instead of a to that's why is not a nuxt-link. 

Fixed it in this PR https://github.com/mesg-foundation/mesg-components/pull/104.

Waiting for you to review it.

**Comment by EmmanuelDrouin** [comment]
Todo:
- [ ] Install new version of the menu (button bug fix)

File: components/Menu.vue (hunk 1 above)
Comment (EmmanuelDrouin) [reply components/Menu.vue]: Done

**Comment by EmmanuelDrouin** [comment]
@antho1404 PR ready for review

**Pull request merged**